packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -99,6 +99,14 @@ export async function execCommand(command: string, options: SpawnOptions = {}) {
   return true;
 }
 
+export async function getNodeBinPath(entrypointDir: string) {
```

Comment:
```suggestion
export async function getNodeBinPath({ cwd : string }) {
```

Let's use an object instead of parameters because we might want to add more than 1 arg in the future.